templates/express/server.js

```diff
@@ -1,3 +1,7 @@
+if (process.env.NODE_ENV === "development") {
+  require("dotenv").config();
```

Comment:
`dotenv` is not a dependency in the package.json file, so it would need to be added. 

This change also looks like a specific use case for you, I'm not sure this will be accepted.

Reply:
We also already have support for `dotenv` in local development in `@remix-run/dev`

Reply:
you can get this same behavior using `node --require dotenv/config server.js` without having to have it as a runtime check, which is what we have set up for the stacks (indie and blues anyway) https://github.com/remix-run/blues-stack/blob/main/package.json#L11

Reply:
We also already have support for `dotenv` in local development in `@remix-run/dev`